sdv/constraints/tabular.py

```diff
@@ -539,6 +541,158 @@ def reverse_transform(self, table_data):
+    applies a sigmoid to the data and then scales it back to the original space.
+
+    Args:
+        constraint_column (str):
+            Name of the column to which the constraint will be applied.
+        low (float or str):
+            If float, lower bound on the values of the ``constraint_column``.
+            If string, name of the column which will be the lower bound.
+        high (float or str):
+            If float, upper bound on the values of the ``constraint_column``.
+            If string, name of the column which will be the upper bound.
+        strict (bool):
+            Whether the comparison of the values should be strict ``>=`` or
+            not ``>`` when comparing them. Currently, this is only respected
+            if ``reject_sampling`` or ``all`` handling strategies are used.
+        handling_strategy (str):
+            How this Constraint should be handled, which can be ``transform``
+            or ``reject_sampling``. Defaults to ``transform``.
+    """
+
+    def __init__(self, column, low, high, strict=False, handling_strategy='transform',
+                 fit_columns_model=True):
+        self.constraint_column = column
+        self._low = low
+        self._high = high
+        self._strict = strict
+        self._transformed_column = f'#{self.constraint_column}#{self._low}#{self._high}'
+        super().__init__(handling_strategy=handling_strategy,
+                         fit_columns_model=fit_columns_model)
+
+    def _get_low(self, table_data):
+        """Return the appropriate lower bound.
+
+        If the ``low`` value was passed as a string, returns the column named ``low``.
+        If it is a float, returns the value itself.
+
+        Args:
+            table_data (pandas.DataFrame):
+                The Table data.
+
+        Returns:
+            pandas.DataFrame or float:
+                The lower bound.
+        """
+        if isinstance(self._low, str):
+            return table_data[self._low]
+
+        return self._low
+
+    def _get_high(self, table_data):
+        """Return the appropriate upper bound.
+
+        If the ``high`` value was passed as a string, returns the column named ``high``.
+        If it is a float, returns the value itself.
+
+        Args:
+            table_data (pandas.DataFrame):
+                The Table data.
+
+        Returns:
+            pandas.DataFrame or float:
+                The upper bound.
+        """
+        if isinstance(self._high, str):
+            return table_data[self._high]
+
+        return self._high
+
+    def is_valid(self, table_data):
+        """Say whether the ``constraint_column`` is between the ``low`` and ``high`` values.
+
+        Args:
+            table_data (pandas.DataFrame):
+                Table data.
+
+        Returns:
+            pandas.Series:
+                Whether each row is valid.
+        """
+        if self._strict:
```

Comment:
Maywe we could use `operator.le` and `operator.lt`?

```python
import operator

def __init__(self, ...strict):
   self._lt = operator.le if strict else operator.le

def is_valid(self, table_data):
    ...
    satisfy_low_bound = self._lt(self._get_low(table_data), table_data[self.constraint_column])
    ...
```

Reply:
That's neat, I didn't know that 👍